### PR TITLE
Draft pick

### PR DESCRIPTION
**Pull request by PolinaRomanchenko**
1) Bug fixed within draft_pick function
2) Tests for input error handling are added
3) Output tests are added

*Note:*
Not sure if we need to make test files for every function, or do it like in Python - all in one. I assumed that `cause some test files were created through "set up repo" process - it expects tests in different files.

### CODE REVIEW

File: R/draft_pick.R [hunk 1]
```diff
@@ -19,13 +22,22 @@
 library(tidyverse)
 library(httr)
 library(jsonlite)
+library(testthat)
```
Comment (jnederlo) [inline R/draft_pick.R]: I think we are supposed to add our functions as imports to the roxygen documentation by doing:

```
#' @import tidyverse
#' @import httr
#' @import jsonlite
#' @import testthat
```

```suggestion
```

Comment (jnederlo) [reply R/draft_pick.R]: then in the directory with the function run this to add the library to the Namespace:

`devtools::document()`

Comment (PolinaRomanchenko) [reply R/draft_pick.R]: Oh, ok. I didn't know that! Thank you!

**Review by jnederlo — APPROVED** [review]
Good work. I'm assuming all tests pass, as the code looks proper. I'm not sure how we should structure the test directory/file either. I'm sure we can adjust it as we work if needed.

**Pull request merged**